scripts/tests/test_validate_docstrings.py

```diff
@@ -1004,6 +1006,32 @@ def test_item_subsection(self, idx, subsection):
         assert result[idx][3] == subsection
 
 
+class TestDocstringClass(object):
+    @pytest.mark.parametrize('name_and_expected_obj',
```

Comment:
Minor stylistic change but you should just be able to do `name, expected` on this line and add `name` and `expected` as separate parameters to the function call instead of unpacking the tuple within your test